src/Aspire.Dashboard/Components/Layout/MainLayout.razor

```diff
@@ -31,7 +31,7 @@
     <FluentDialogProvider />
     <FluentTooltipProvider />
     <div id="blazor-error-ui">
-        @Loc[nameof(Layout.MainLayoutUnhandledErrorMessage)]
+        <span id="error-message">@Loc[nameof(Layout.MainLayoutUnhandledErrorMessage)]</span>
```

Comment:
I don't see any references to the id `error-message` elsewhere in the code. Is it used by blazor internally? Or can the id be removed?

Reply:
I had thought it would be a good idea to explicitly state its purpose, removed :)